File: app/src/main/java/com/aslansari/hypocoin/repository/restapi/Status.java
package com.aslansari.hypocoin.repository.restapi;

import com.google.gson.annotations.SerializedName;

public final class Status {
    @SerializedName(value = "elapsed")
    public int elapsed;
    @SerializedName(value = "timestamp")
    public String timestamp; // TODO iso time stamp GSON converter
}

File: app/src/main/java/com/aslansari/hypocoin/repository/restapi/Status.kt
package com.aslansari.hypocoin.repository.restapi

import com.google.gson.annotations.SerializedName

class Status {
    @SerializedName(value = "elapsed")
    var elapsed = 0

    @SerializedName(value = "timestamp")
    var timestamp // TODO iso time stamp GSON converter
            : String? = null
}